### ISSUE TITLE
Help us undo our mistakes (CTRL-Z)

### ISSUE THREAD

**Issue by pieeetr** [Cinteros/FetchXMLBuilder#76]
With a tool so great as the FetchXMLBuilder one would think it'd be impossible to make mistakes using it. However, that's not the case. It is possible and I make them quite frequently. First of all, I'd like to make it clear that I'm not saying that the punishments for doing these errors are unfair. But I'm standing here on my knees asking for a feature to help us who do bad things, to make it easier to move on with our lives.

Please give us the power to undo (our mistakes).

**Comment by shytikov** [comment]
:+1:

**Comment by shytikov** [comment]
Mistakes shall not pass!

**Comment by rappen** [comment]
For testing purposes, please download the following zip file, unblock it, unpack it to your XTB folder.
http://rappen.net/FXB/preview/FXB-undo-preview-1062.zip

__Feedback requested!__

**Comment by pieeetr** [comment]
Works like a charm @rappen. Many thanks. A "nice to have" would be if you could do the same without the whole section of the query reloading, but that's just if it's easy to fix.

**Comment by rappen** [comment]
Improved version/build 1063:
http://rappen.net/FXB/preview/FXB-undo-preview-1063.zip
